Algorithms/Sort/InsertionSort/InsertionSort.cpp

```diff
@@ -0,0 +1,53 @@
+#include <iostream>
+#include <vector>
+
+/**
+ * @brief	Perform Insertion Sort on a vector.
+ *
+ * This function sorts the elements of the vector in ascending order using the
+ * Insertion Sort algorithm.
+ *
+ * @param	array	Reference to the vector to be sorted.
+ */
+void insertionSort(std::vector<int>& array) {
+	// This is the length of given array
+	int arrayLength = array.size();
+
+	// Iterating through each element of the array from index 1 to lastIndex(arrayLength -1)
```

Comment:
We will be stopping over commenting now on. The example is getting filled with comments (about 30% at minimum). We should write code that is self explanatory and don't require comment to explain.

Use as less comment possible.

> [!Note]
> The doc comment are good. And won't need changing.